Makefile

```diff
@@ -143,17 +135,17 @@ cross-platform: darwin linux windows ## Build all distributable (cross-platform)
 
 darwin: $(BIN_DARWIN) ## Build for Darwin (macOS)
 
-$(BIN_DARWIN): pkged.go
+$(BIN_DARWIN): zz_filesystem_generated.go
```

Comment:
Definitely prefer this name change.

Reply:
@lance  Like it's ok, or you would rename it?
`zz_` is just prefix to push it down in diffs.

Reply:
It's OK - I like it better than `pkged.go` for that reason.

> @lance Like it's ok, or you would rename it? `zz_` is just prefix to push it down in diffs.